saleor/graphql/order/tests/mutations/test_order_confirm.py

```diff
@@ -103,6 +106,53 @@ def test_order_confirm(
     )
 
 
+@patch("saleor.graphql.order.mutations.order_confirm.request_charge_action")
+def test_order_confirm_charge_transaction_items(
```

Comment:
IMHo, you should add three more tests here.
One with multiple transaction_items, where sum of authorized_value is equal to order.total
Second one with single transaction_item which has authorized_value bigger than order.total(to confirm that we always charge maximally `order.total` value)
Third one with multiple transaction_items, where sum of authorized_value is bigger than order.total.